src/components/SelectableMainTable.tsx

```diff
@@ -110,7 +150,9 @@ const SelectableMainTable: FC<Props> = ({
             }
             labelClassName="u-no-margin--bottom"
             checked={isRowSelected}
-            onChange={toggleRow}
+            onChange={(event: PointerEvent<HTMLInputElement>) =>
```

Comment:
I think we can keep this as it was before. In the `toggleRow` function, we can directly access `rowIndex` from the parent scope and the event will be automatically forwarded as first parameter.